.changeset/angry-candles-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```

Comment:
This isn't breaking, right? Seems like this could be `patch` (since it's on 0.x still)

Reply:
Done, changed to `patch` 👍